modules/builds-configuring-post-commit-build-hooks.adoc

```diff
@@ -10,42 +10,48 @@ There are different ways to configure the post build hook. All forms in the foll
 
 .Procedure
 
-* Shell script:
+* Do one of the following options:
 +
+|===
+|Option |Description
+
+|Shell script
+a|
 [source,yaml]
 ----
 postCommit:
   script: "bundle exec rake test --verbose"
 ----
-+
-The `script` value is a shell script to be run with `/bin/sh -ic`. Use this when a shell script is appropriate to execute the build hook. For example, for running unit tests as above. To control the image entry point, or if the image does not have `/bin/sh`, use `command` and/or `args`.
-+
+
+The `script` value is a shell script to be run with `/bin/sh -ic`. Use this when a shell script is appropriate to execute the build hook. For example, for running unit tests as above. To control the image entry point, or if the image does not have `/bin/sh`, use `command`, or `args`, or both.
```

Comment:
```suggestion
The `script` value is a shell script to be run with `/bin/sh -ic`. Use this when a shell script is appropriate to execute the build hook. For example, for running unit tests as above. To control the image entry point or if the image does not have `/bin/sh`, use `command`, or `args`, or both.
```

Reply:
I would favor the oxford comma and add that you should clarify with nouns after `command` and `arg`. If you are referring to commands and arguments in general I don't think you need ` `.  This is pretty far out of my area though so I leave to you.